impress-org/givewp
# Feature: add link to PDF receipt in confirmation receipt

## Description



This PR adds a link to the PDF receipt on the confirmation page only if the PDF receipts addon is active. 

## Affects



The confirmation/receipt page.

## Visuals



![download-pdf](https://github.com/impress-org/givewp/assets/16308864/1e85a157-c23f-4839-9b0c-33e12ec6934f)

## Testing Instructions

1. Active the PDF receipts addon and switch to this branch: https://github.com/impress-org/give-pdf-receipts/pull/249;
2. Create a donation using a v3 form;
3. Check the link to the PDF receipt at the end of the confirmation/receipt page.



## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205336489321291
